release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2553,6 +2553,8 @@ configmap/gateway-mode-config created
 
 For additional guidance, see (link:https://access.redhat.com/solutions/6962127[*KCS*]) and (link:https://bugzilla.redhat.com/show_bug.cgi?id=2089389[*BZ#2089389*]). This setting will be addressed further in a future release.
 
+* It is not possible to create a macvlan on the physical function (PF) when a virtual function (VF) already exists. This issue affects the Intel E810 NIC. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2120585[*BZ#2120585*])
```

Comment:
Same thing not sure about spelling, MacVLan or macvlan.

Reply:
@JoeAldinger  I did a search on docs before adding to see what it is, and it seems to be macvlan across the docs. I think we are OK with macvlan as I think it well understood this is the CNI plugin original content already merged to 4.12 RN so would like to keep this the same.